mifiel.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
-  spec.required_ruby_version = '~> 2.3'
+  spec.required_ruby_version = '>= 2.3'
 
   spec.add_runtime_dependency 'activesupport'
   spec.add_runtime_dependency 'api-auth', '~> 1.4'
```

Comment:
It would be nice to upgrade this dependency since it is using deprecated methods:

```
WARNING: `Faraday::Request#method` is deprecated; use `#http_method` instead. It will be removed in or after version 2.0.
`Faraday::Request#method` called from /Users/daniel/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/api-auth-1.5.0/lib/api_auth/request_drivers/faraday.rb:31:in `populate_content_md5'
```

Reply:
@genaromadrid :)

Reply:
Agreed, we have a restriction there in the application side but i will check if we can upgrade it here and configure it so it doesn't break with the application.